pkg/systemd/logs.html

```diff
@@ -32,45 +32,46 @@
-        <button class="btn btn-default dropdown-toggle" data-toggle="dropdown">
-          <span id="journal-current-day"></span>
-          <i class="fa fa-caret-down pf-c-context-selector__toggle-icon" aria-hidden="true"></i>
-        </button>
-        <ul class="dropdown-menu" role="menu">
-          <li><a role="link" tabindex="0" data-op="recent" translate="yes">Recent</a></li>
-          <li><a role="link" tabindex="0" data-op="boot" translate="yes">Current boot</a></li>
-          <li><a role="link" tabindex="0" data-op="previous-boot" translate="yes">Previous boot</a></li>
-          <li><a role="link" tabindex="0" data-op="last-24h" translate="yes">Last 24 hours</a></li>
-          <li><a role="link" tabindex="0" data-op="last-week" translate="yes">Last 7 days</a></li>
-        </ul>
+      <div class="filter">
+          <label for="journal-current-day-menu" hidden translate>Time Limit</label>
+          <select id="journal-current-day-menu" class="ct-select">
+            <option value="recent" translate>Recent</option>
+            <option value="boot" translate>Current boot</option>
+            <option value="previous-boot" translate>Previous boot</option>
+            <option value="last-24h" translate>Last 24 hours</option>
+            <option value="last-week" translate>Last 7 days</option>
+          </select>
       </div>
-      <span class="" translate>Severity</span>
-      <div class="btn-group" id="journal-prio-menu">
-        <button class="btn btn-default dropdown-toggle" data-toggle="dropdown">
-          <span id="journal-prio"></span>
-          <i class="fa fa-caret-down pf-c-context-selector__toggle-icon" aria-hidden="true"></i>
-        </button>
-        <ul id="prio-lists" class="dropdown-menu" role="menu">
-          <li><a role="link" tabindex="0" data-prio="*" translate="yes">Everything</a></li>
-          <li><a role="link" tabindex="0" data-prio="0" translate="yes">Only Emergency</a></li>
-          <li><a role="link" tabindex="0" data-prio="1" translate="yes">Alert and above</a></li>
-          <li><a role="link" tabindex="0" data-prio="2" translate="yes">Critical and above</a></li>
-          <li><a role="link" tabindex="0" data-prio="3" translate="yes">Error and above</a></li>
-          <li><a role="link" tabindex="0" data-prio="4" translate="yes">Warning and above</a></li>
-          <li><a role="link" tabindex="0" data-prio="5" translate="yes">Notice and above</a></li>
-          <li><a role="link" tabindex="0" data-prio="6" translate="yes">Info and above</a></li>
-          <li><a role="link" tabindex="0" data-prio="7" translate="yes">Debug and above</a></li>
-        </ul>
+      <span class="filters-toggle" hidden>
+        <button class="link-button" translate>Show all filters</button>
+      </span>
+      <div class="filters-toggle full-row" hidden></div>
+      <div class="filter">
+          <label for="journal-prio-menu" translate>Severity</label>
```

Comment:
I wonder if this should be renamed to _Priority_, since that's the string we put in the text filter. A little weird if they are inconsistent with each other for no apparent reason.

Reply:
Done